html/admin/publicSites/embed/asset.php

```diff
@@ -1,5 +1,5 @@
 <?php
```

Comment:
Not sure about the url adam-rms.com/publicSites/embed i think adam-rms.com/public/embed would be better or even adam-rms.com/public/

Reply:
I'm not expecting anyone to see them, so we should do whatever is best for folder structure. At the moment we have nothing that's more than 1 word so whichever people think is best

Reply:
Think public/embeds is best as they're technically no longer sites themselves and we potentially may do more in the future with public stuff?